queries/haskell/injections.scm

```diff
@@ -0,0 +1,63 @@
+;; ----------------------------------------------------------------------------- 
+;; General language injection
+
+(quasiquote 
+ ((quoter) @language)
+ ((quasiquote_body) @content)
+)
+
+
+;; ----------------------------------------------------------------------------- 
+;; shakespeare library
+;; NOTE: doesn't support templating
+
+; CoffeeScript: Text.Coffee
+(quasiquote 
+ (quoter) @_name
+ (#eq? @_name "coffee")
+ ((quasiquote_body) @content)
+ (#set! language "coffeescript")
+)
+
+; CSS: Text.Cassius, Text.Lucius
+(quasiquote 
+ (quoter) @_name
+ (#any-of? @_name "cassius" "lucius")
+ ((quasiquote_body) @content)
+ (#set! language "css")
+)
+
+; HTML: Text.Hamlet
+(quasiquote 
+ (quoter) @_name
+ (#any-of? @_name "shamlet" "xshamlet" "hamlet" "xhamlet" "ihamlet")
+ ((quasiquote_body) @content)
+ (#set! language "html")
```

Comment:
is there any reason to use `(#set! language "html")` instead of just `((quasiquote_body) @html)` ?

also, I think we need to remove the trailing whitespaces